.gitpod.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+tasks:
+  - init: |
+      mkdir -p cmake-build-debug-gitpod
+      cd cmake-build-debug-gitpod && ../container/support-scripts/build-all.sh && make -j
+
+image:
+  file: container/Containerfile.gitpod
```

Comment:
I think it would be good to have an actual image here (see the `container/README.md` for info on how to build and deploy) instead of having to build the image every time.

I know INFRA does provide access to Docker (and I think also to GHCR), and I am willing to work with them on setting up a location for pushing this dev image.

This would then be changed to e.g.:
```yml
image: docker.io/apache/celix-dev:20231010-gitpod
```

Reply:
Sticking to `file` for now, since otherwise deployment and versioning is also something to take into account.